src/components/SingleDatePickerInput.jsx

```diff
@@ -217,7 +217,10 @@ export default withStyles(({ reactDates: { color } }) => ({
   },
 
   SingleDatePickerInput__withBorder: {
-    border: `1px solid ${color.core.border}`,
+    borderColor: color.border,
+    borderWidth: border.pickerInput.borderWidth,
+    borderStyle: border.pickerInput.borderStyle,
+    borderRadius: border.pickerInput.borderRadius,
```

Comment:
Maybe this is not needed? Not sure

Reply:
def is. thank you for doing it in both pickers!